components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -114,7 +114,7 @@ function TeamUsage() {
     const calculateTotalUsage = () => {
         let totalCredits = 0;
         billedUsage.forEach((session) => (totalCredits += session.credits));
-        return totalCredits.toFixed(2);
+        return totalCredits.toLocaleString(undefined, { maximumFractionDigits: 2 });
```

Comment:
I think this was meant to be fixed, but was not part of those discussions.

Deferring review to @gtsiolis :basketball: